pkg/testutils/sensors/load.go

```diff
@@ -132,6 +134,12 @@ func mergeInBaseSensorMaps(sensorMaps []SensorMap, sensorProgs []SensorProg, t *
 		SensorMap{Name: "execve_val", Progs: []uint{2}},
 	}
 
+	if kernels.EnableLargeProgs() {
+		baseMaps = append(baseMaps, SensorMap{Name: "tcpmon_map", Progs: []uint{0, 1, 2, 3}})
+	} else {
+		baseMaps = append(baseMaps, SensorMap{Name: "tcpmon_map", Progs: []uint{1, 2, 3}})
+	}
+
```

Comment:
Why is this needed?

My reading is:
Program 0 is event_execve, but I don't see how event_execve will ever need tcpmon_map since we seem to be alwats tail-calling into the program that sends the event (execve_send). Am I missing something?

Reply:
there's the data interface that sends events and is part of event_execve,
but it's only compiled in for large programs.. it's still not there for 4.19

there's PR for that alreay, but I wanted to split the execve event first